crane-cloud/frontend
# Chore: Profile summary, integrate backend API

# Description

Show Profile summary with actual backend data

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

https://trello.com/c/aHsIhf8U

## How Can This Been Tested?

Depends on the backend PR 
https://github.com/crane-cloud/backend/pull/512

When merged, check out the profile summary page

## Checklist:


# Screenshots

<img width="1469" alt="Screenshot 2024-07-03 at 15 48 20" src="https://github.com/crane-cloud/frontend/assets/69305400/473f06e5-19cf-4f14-99a0-2246f77877cf">
